.github/workflows/integration_test.yaml

```diff
@@ -159,152 +159,25 @@ jobs:
-          CHAOS_NAMESPACE: ${{ inputs.chaos-namespace }}
-          DELAY: ${{ inputs.chaos-status-delay }}
-          DURATION: ${{ inputs.chaos-status-duration }}
-          EXPERIMENT_NAME: ${{ inputs.chaos-experiments }}
-          TOTAL_CHAOS_DURATION: ${{ inputs.chaos-duration }}
-          KUBE_CONFIG_DATA: ${{ env.KUBE_CONFIG_DATA }}
-      - name: Install k6s
-        if: ${{ inputs.load-test-enabled }}
-        run: sudo snap install k6
-      - name: Run k6 load tests
-        if: ${{ inputs.load-test-enabled }}
-        run: k6 run load_tests/load-test.js ${{ inputs.load-test-run-args }}
-      - name: Run Github Trivy FS Action
-        if: ${{ inputs.trivy-fs-enabled }}
-        uses: aquasecurity/trivy-action@master
-        with:
-          scan-type: 'fs'
-          scan-ref: ${{ inputs.trivy-fs-ref }}
-          trivy-config: ${{ inputs.trivy-fs-config }}
-      - name: Set Zap target env for Github Zap Action to Juju Unit IP Address
-        if: ${{ inputs.zap-enabled && inputs.zap-target == '' }}
-        run: echo "ZAP_TARGET=$(juju show-unit ${{ env.CHARM_NAME }}/0 --format=json | jq -r '.["${{ env.CHARM_NAME }}/0"]["address"]')" >> $GITHUB_ENV
-      - name: Set Zap target env for Github Zap Action to zap-target value
-        if: ${{ inputs.zap-enabled && inputs.zap-target != '' }}
-        run: echo "ZAP_TARGET=${{ inputs.zap-target }}" >> $GITHUB_ENV
-      - name: Run command before Github Zap Action
-        if: ${{ inputs.zap-enabled && inputs.zap-before-command != '' }}
-        run: ${{ inputs.zap-before-command }}
-        env:
-          ZAP_TARGET: ${{ env.ZAP_TARGET }}
-      - name: Run Github Zap Action
-        if: ${{ inputs.zap-enabled }}
-        uses: zaproxy/action-baseline@v0.7.0
-        env:
-          ZAP_AUTH_HEADER:  ${{ inputs.zap-auth-header }}
-          ZAP_AUTH_HEADER_VALUE:  ${{ inputs.zap-auth-header-value  }}
-        with:
-          issue_title: 'OWASP ZAP report'
-          fail_action: false
-          target: ${{ inputs.zap-target-protocol }}://${{ env.ZAP_TARGET }}:${{ inputs.zap-target-port }}/
-          cmd_options: ${{ inputs.zap-cmd-options }}
-          rules_file_name: ${{ inputs.zap-rules-file-name }}
+    with:
+      extra-arguments: ${{ inputs.extra-arguments }}
+      extra-test-matrix: ${{ inputs.extra-test-matrix }}
+      images: ${{ needs.build-images.outputs.images }}
+      modules: ${{ inputs.modules }}
+      owner: ${{ github.repository_owner }}
+      pre-run-script: ${{ inputs.pre-run-script }}
+      provider: ${{ inputs.provider }}
+      registry: ghcr.io
+      runs-on: ${{ needs.get-runner-image.outputs.runs-on }}
+      series: ${{ inputs.series }}
+      setup-devstack-swift: ${{ inputs.setup-devstack-swift }}
+      trivy-fs-config: ${{ inputs.trivy-fs-config }}
+      trivy-fs-enabled: ${{ inputs.trivy-fs-enabled }}
+      trivy-fs-ref: ${{ inputs.trivy-fs-ref }}
+      trivy-image-config: ${{ inputs.trivy-image-config }}
```

Comment:
As discussed privately, all the inputs of `integration_test_run.yaml` must be also passed there as they aren't inherited